### PR TITLE
WiP: Make a new project for my CV version 3

### PR DESCRIPTION
**Pull request by Nicolleta1981**

### CODE REVIEW

**Review by biggicode — APPROVED** [review]
Well done! 👍

File: web/nicolleta1981/my cv-v3-starter/css/grid.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+@media screen and (min-width: 1024px) {
+  .grid {
+    grid-template-columns: repeat(15, 1fr);
+    grid-gap: 20px;
+    margin: 0 150px;
+  }
+}
+
+[data-column="5"] {
+  grid-column:span 5;
```
Comment (biggicode) [inline web/nicolleta1981/my cv-v3-starter/css/grid.css:25]: Add a space

**Review by vasilemaftita — APPROVED** [review]
Nice; Work!

**Review by vasilemaftita — APPROVED** [review]
Nice; Work!

**Review by Adrian01C — COMMENTED** [review]
Nicee

**Review by vasilemaftita — APPROVED** [review]
Nice; Work...

**Review by psyhedeliq — APPROVED** [review]

**Review by simonamarcu2 — APPROVED** [review]
Great job!

**Review by Adrian01C — APPROVED** [review]
nice

**Pull request merged**